source/ConfigFiles.h

```diff
@@ -69,6 +69,11 @@ class StartupConfig {
     String userConfig() {
         return userConfigPath + "userconfig.Any";
     }
+	
+    /** filename with given path to user status file */
+	String userStatusConfig() {
+		return userConfigPath + "userstatus.Any";
```

Comment:
I.e. replace `userConfigPath` with `userConfigFile`?

This changes is needed to support testing with separate configs. Most of the tests need to be run in a clean environment and it would be a pain if every little change made to the main experiment configs kept breaking the tests.

I made a single separate test case to validate the main configs allow the app to run.

Reply:
I may have been confused here. I think the "right" thing to do is let the experiment designer specify a full path for the user status/config independently without any file system tie between the two (i.e. co-location in a single directory). This way we could name our test files `test_userstatus.Any` and not conflict with a default `userstatus.Any` file.

An example motivating case would be something like a set of experiments hosted through an NFS where the designer might want a `userConfigs` directory and a `userStatus` directory each containing multiple user config/status files.

If this doesn't apply here, we can always take the changes as-is and create a new issue to implement this as a standalone feature request.